src/libsync/filesystem.cpp

```diff
@@ -235,8 +262,12 @@ bool FileSystem::uncheckedRenameReplace(const QString& originFileName,
 
 #else //Q_OS_WIN
     BOOL ok;
-    ok = MoveFileEx((wchar_t*)originFileName.utf16(),
-                    (wchar_t*)destinationFileName.utf16(),
+    QString orig = longWinPath(originFileName);
+    QString dest = longWinPath(destinationFileName);
+
+    qDebug() << "** MOVE: " << orig;
```

Comment:
This qDebug seems to lack a bit of context, could you either make it useful for production or remove it?